apollo-gradle-plugin/src/main/groovy/com/apollostack/android/gradle/ApolloIRGenTask.groovy

```diff
@@ -40,7 +42,8 @@ public class ApolloIRGenTask extends NodeTask {
     dependsOn(ApolloCodeGenInstallTask.NAME)
 
     possibleGraphQLPaths = buildPossibleGraphQLPaths()
-    outputDir = project.file("src/$variant/graphql/generatedIR")
+    schemaFile = userProvidedSchemaFile() ?: searchForSchemaFile()
+    outputDir = schemaFile.getParentFile()
```

Comment:
Should we be generating the IR under the `build` folder? For now, it's generated under the same folder as the schema file.